src/main/java/net/minecraftforge/common/config/Configuration.java

```diff
@@ -80,9 +89,16 @@
 
     public Configuration(){}
 
+    @Deprecated
+    public Configuration(CharSource staticSource, CharSink staticSink){
```

Comment:
Not completely. The problem is that we should load from parent if configuration is loaded from file, we should not in all other cases. Because if we sync configuration accross net and load from local file, this makes no sense.
Although, i'll probably tweek it now.